qutebrowser/browser/network/networkmanager.py

```diff
@@ -241,12 +243,31 @@ def clear_rejected_ssl_errors(self, url):
     @pyqtSlot('QNetworkReply', 'QAuthenticator')
     def on_authentication_required(self, reply, authenticator):
         """Called when a website needs authentication."""
-        answer = self._ask("Username ({}):".format(authenticator.realm()),
-                           mode=usertypes.PromptMode.user_pwd,
-                           owner=reply)
-        if answer is not None:
-            authenticator.setUser(answer.user)
-            authenticator.setPassword(answer.password)
+        user, password = None, None
+        if not hasattr(reply, "netrc_used"):
+            try:
+                reply.netrc_used = True
+                net = netrc.netrc()
+                authenticators = net.authenticators(reply.url().host())
+                if authenticators is not None:
+                    # pylint: disable=unpacking-non-sequence
+                    (user, _account, password) = authenticators
+            except FileNotFoundError:
+                log.misc.info("No .netrc file found")
```

Comment:
Sorry for just noticing this now - please make this `debug`. For people not using `.netrc` (pretty much everyone?) it might be confusing to have this logged for every authentication.